.github/workflows/build-lint-test.yml

```diff
@@ -149,9 +163,9 @@ jobs:
       - name: Download backend-build
         uses: actions/download-artifact@v4
         with:
-          name: backend-build
+          name: backend-build-${{ matrix.os }}
           path: builds/backend
       - name: make backend build executable
-        run: chmod a+x ../builds/backend/abacus
+        run: chmod a+x ../builds/backend/${{ runner.os == 'Windows' && 'abacus.exe' || 'abacus' }}
```

Comment:
It's using PowerShell, not sure why `chmod` works on GitHub Actions because it does error out on my Windows VM.

![Screenshot 2024-11-21 at 16 44 08](https://github.com/user-attachments/assets/61e1e788-6a8a-4be2-a4fc-9393b7f80f48)

In 863b998721709a27efa5823508051e3973477013 I added a conditional to skip this step on Windows, and the Playwright test still works fine.